torch/quantization/_numeric_suite.py

```diff
@@ -115,6 +118,22 @@ def forward(self, x, y):
             self.stats["float"] = torch.cat((self.stats["float"], y.detach()))
 
 
+class TensorLogger(Logger):
```

Comment:
Couldnt leave a comment for RecordingLogger above: Why not keep the behavior of both Tensor and RecordingLoggers consistent. Both should not dequantize tensors, that way they just do simple capture of the data.

Reply:
That's a good point, it's better to keep quantized tensor in both loggers.